C/services/filter-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -221,8 +229,10 @@ PLUGIN_HANDLE filter_plugin_init_fn(ConfigCategory* config,
  */
 void *PluginInterfaceInit(const char *pluginName, const char * pluginPathName)
 {
-        // Set plugin name
-        sPluginName = pluginName;
+        // Set plugin name for common-interfaces
+        gPluginName = pluginName;
```

Comment:
South plugin interface code also writes gPluginName, so it would be over-written here?

Reply:
this variable gets the right value depending on the cpp file that sets it